hands-on-exercise/3-cosmjs-adv/5-server-side.md

```diff
@@ -587,12 +587,6 @@ const handleEvents = async (events: StringEvent[]): Promise<void> => {
 }
 ```
 
-This needs a new import:
-
-```typescript [https://github.com/cosmos/academy-checkers-ui/blob/server-indexing/src/server/indexer.ts#L4]
-import { ABCIMessageLog, Attribute, StringEvent } from "cosmjs-types/cosmos/base/abci/v1beta1/abci"
```

Comment:
It was placed here because `StringEvent` is on line 576. And we added all imports together instead of updating it later.
Perhaps it is better to keep only `import StringEvent`, and then adjust it below where you added it, by making it a `diff-typescript`.

Reply:
Oh